### DISCUSSION TITLE
Add MacPorts install info

### DISCUSSION THREAD

**Discussion by herbygillot** [sayanarijit/xplr#24] [Show and tell]
Can be added to the wiki, but on macOS, `xplr` can now be installed via [MacPorts](https://www.macports.org):

1. Install MacPorts
2. Run:
    ```
    sudo port selfupdate
    sudo port install xplr
    ```

**Comment by sayanarijit** [comment]
Hey thanks.

**Reply by herbygillot** [reply]
Thank you.  Can we embed this info into the wiki instead of having a link to this discussion? Up to you either way.

**Reply by sayanarijit** [reply]
Sure if that's what you want. I wanted people to know that it's your contribution and you are maintainer. Are you sure we should embed this in the wiki?

**Reply by herbygillot** [reply]
Ah OK, I appreciate that.  I was trying to minimize the amount of page navigation, but let's leave things as-is then.  Thank you @sayanarijit.

**Comment by sayanarijit** [comment]
@herbygillot There's a critical update: https://github.com/sayanarijit/xplr/releases/tag/v0.5.10

**Reply by herbygillot** [reply]
> @herbygillot There's a critical update: https://github.com/sayanarijit/xplr/releases/tag/v0.5.10

Updated: https://github.com/macports/macports-ports/commit/ad94963620cd1831179d6a962a1718ad11614c6c
This will take a bit of time to build and distribute through MacPorts mirrors.

**Reply by sayanarijit** [reply]
Thanks :+1: .

**Reply by herbygillot** [reply]
Thanks for the heads-up
